[18.12.2024  8:39:44,14] Running innvandrere_bosatt.py 
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
X_FUNCTIONS_KEY loaded successfully.
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
GITHUB_TOKEN loaded successfully.
Bosatt JSON data loaded successfully.
[18.12.2024 08:39:47,654 Deleted file from temp folder: andel_innvandrere_bosatt.csv

Traceback (most recent call last):
  File "D:\Scripts\analyse\Telemark\Python\Queries\09_Innvandrere_og_inkludering\Innvandrerbefolkningen\innvandrere_bosatt.py", line 121, in <module>
    is_new_data = compare_to_github(df, file_name, github_folder, temp_folder)
  File "D:\Scripts\analyse\Telemark\Python\Helper_scripts\github_functions.py", line 173, in compare_to_github
    existing_data = pd.read_csv(StringIO(existing_file_content), encoding='utf-8')
TypeError: initial_value must be str or None, not DataFrame
[18.12.2024  8:39:47,76] Script failed with error code 1 
